resolwe_bio/processes/import_data/annotation.yml

```diff
@@ -171,7 +204,21 @@
     program: |
       re-import {{ src.file_temp }} {{ src.file }} "gtf|gff" "gtf" 0.3
 
+      igvtools sort "${NAME}".gtf "${NAME}"_sorted.gtf
+      re-checkrc "Sorting of GTF file failed."
+      igvtools index "${NAME}"_sorted.gtf || re-warning "Indexing of GTF file failed."
+
+      flatfile-to-json.pl --gff "${NAME}"_sorted.gtf --out . --trackLabel "annotation"
+      re-checkrc "Annotation track processing for JBrowse failed."
+
       re-save-file annot "${NAME}.gtf"
+      re-save-file annot_sorted "${NAME}"_sorted.gtf
+      re-save-file annot_sorted_track_jbrowse trackList.json tracks/annotation
+
+      if [ -f "${NAME}"_sorted.gtf.idx ]; then
```

Comment:
Did you write a test for this case?

Reply:
It is true, that in this case we wouldn't notice if it fails. I have added a test ``self.assertFields(upload_gtf, 'annot_sorted_idx_igv', {'file': 'Hs GRCh38_86 upload_sorted.gtf.idx', 'total_size': 116})``